README.md

```diff
@@ -20,10 +20,15 @@ What the Backup Restore Operator is not:
 - Configured to back up user-created resources on the Rancher cluster.
 
 ### Branches and Releases
+This is the current branch strategy for `rancher/shell`, it may change in the future.
```

Comment:
Shouldn't this be _This is the current branch strategy for `rancher/backup-restore-operator`, it may change in the future._?

Reply:
sloppy copy pasta. good catch